tests/integration/test_flatten.py

```diff
@@ -67,11 +35,33 @@ def run_test(np_arr, num_arr):
 
 
 # test ndarray.flatten w/ 1D, 2D and 3D arrays
+@pytest.mark.parametrize("order", ["C", "F", "A"], ids=str)
 @pytest.mark.parametrize("size", SIZES, ids=str)
-def test_basic(size):
+def test_basic(order, size):
     a = np.random.randint(low=0, high=100, size=size)
-    b = convert_to_cunumeric_ndarray(a)
-    run_test(a, b)
+    print_msg = f"np & cunumeric.ndarray.flatten({order})"
+    check_array_method(
+        "flatten", [order], {}, print_msg, ndarray_func=True, ndarray_np=a
+    )
+
+
+class TestNdarrayFlattenErrors:
+    def setup(self):
+        size_a = (1, DIM)
+        anp = np.random.randint(low=0, high=100, size=size_a)
+        self.anum = num.array(anp)
+
+    def test_non_string_order(self):
+        order = 0
```

Comment:
0 is often a special case, suggest also checking non-zero (pos and neg) ints and floats, as well as other negative cases, especially since they all have customized error messages
```python
In [5]: a.flatten(1.3)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In [5], line 1
----> 1 a.flatten(1.3)

TypeError: order must be str, not float

In [6]: a.flatten([1])
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In [6], line 1
----> 1 a.flatten([1])

TypeError: order must be str, not list

```

Reply:
Cased added. Thanks.